content/applications/productivity/mail_plugins/gmail.rst

```diff
@@ -2,94 +2,162 @@
 Gmail Plugin
 ============
 
-Configuration
-=============
+The *Gmail Plugin* integrates an Odoo database with a Gmail inbox, so users can keep track of all
+their work between Gmail and Odoo, without losing any information.
 
-The Gmail :doc:`Mail Plugin <../mail_plugins>` needs to be configured both on Odoo and Gmail.
+Odoo Online users
+=================
 
-.. _mail-plugin/gmail/enable-mail-plugin:
+For databases hosted on Odoo Online (or Odoo.sh), follow the steps below to configure the Gmail 
+Plugin.
 
-Enable Mail Plugin
-------------------
+Install the Gmail Plugin
+------------------------
 
-First, you need to enable the *Mail Plugin* feature in your database. Go to :menuselection:`Settings
---> General Settings --> Integrations`, enable *Mail Plugin*, and *Save* the configuration.
+First, log in to the Gmail account that the user wishes to connect to Odoo.
 
-.. _mail-plugin/gmail/install-plugin:
+From the Gmail inbox, click the plus sign icon on the right side panel to get add-ons. If the side
+panel is not visible, click on the arrow icon at the bottom right corner of the inbox to reveal it.
 
-Install the Gmail Plugin
-------------------------
+.. image:: gmail/gmail-side-panel.png
```

Comment:
This image is too big.

- Capture screenshot in HD (browser at ~1280x600px with ~120% zoom)
- Resize file to tablet/low-res desktop breakpoint b/w 768 and ~1024px (lowest you can go without having to squint at the content).
- Compress file using `pngquant` 
- Replace file locally using `git rm` and `git add`